Packs/SymantecEndpointSecurity/Integrations/SymantecEndpointSecurity/SymantecEndpointSecurity.py

```diff
@@ -0,0 +1,481 @@
+import itertools
+import demistomock as demisto
+from urllib3 import disable_warnings
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+from datetime import datetime
+import dateparser
+import time
+
+disable_warnings()
+
+
+# CONSTANTS
+VENDOR = "symantec"
+PRODUCT = "endpoint_security"
+DEFAULT_CONNECTION_TIMEOUT = 30
```

Comment:
why 30?

Reply:
This is also a recommendation from Symantec.